pkg/controller/machineset/controller.go

```diff
@@ -158,41 +197,42 @@ func (c *MachineSetControllerImpl) createMachine(machineSet *v1alpha1.MachineSet
 	return machine
 }
 
-// getMachines returns a list of machines that match on machineSet.Spec.Selector
-func (c *MachineSetControllerImpl) getMachines(machineSet *v1alpha1.MachineSet) ([]*v1alpha1.Machine, error) {
-	selector, err := metav1.LabelSelectorAsSelector(&machineSet.Spec.Selector)
-	if err != nil {
-		return nil, err
-	}
-	filteredMachines, err := c.machineLister.List(selector)
-	if err != nil {
-		return nil, err
-	}
-	return filteredMachines, err
-}
-
-func shouldAdopt(machineSet *v1alpha1.MachineSet, machine *v1alpha1.Machine) bool {
```

Comment:
Why get rid of the shouldAdopt method? I thought it make it more readable.

Reply:
The reason that I got rid of the separate method was because it duplicated most of the logic for filtering out machines that is irrelevant. Effectively shouldAdopt is the same logic as the do I ignore these machines with the exceptionof if it has an owner ref or not. 
Was trying to optimize it so I didn't need to iterate through the list of machines twice.